### PR TITLE
AK -> CCS Kafka Sync (13 Aug 2020) 

### PR DESCRIPTION
**Pull request by rajinisivaram**
Conflicts:
  - build.gradle (log4j dependencySubstitution)

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by rajinisivaram** [comment]
@niteshmor There was a conflict in the merge from upstream because of https://github.com/confluentinc/kafka/commit/f59086a0a68ba7060d6fdc62730fd1eca6bfa439. Can you review the conflicting change in build.gradle please?

File: build.gradle [hunk 1]
```diff
@@ -119,6 +114,9 @@ allprojects {
             libs.nettyHandler,
             libs.nettyTransportNativeEpoll
           )
+          dependencySubstitution {
+            substitute module("log4j:log4j:1.2.17") because "we use a custom version with security patches" with module("io.confluent:confluent-log4j:1.2.17-cp2")
+          }
         }
```
Comment (niteshmor) [inline build.gradle:120]: @rajinisivaram  My immediate reaction is that this `dependencySubstitution` block is conditional on `name != "zinc"`, which seems to be not what we'd like. We'd like this module substitution regardless. But then, I am not too familiar with zinc (and scala ecosystem in general). 
Is there a quick way for me to build in a way where `name == "zinc"`? I'll be happy to test.

Comment (rajinisivaram) [reply build.gradle:120]: @niteshmor Thanks for the review. Zinc is the scala incremental compiler, I don't think `dependencySubstitution` needs to apply to that. @ijuma Can you confirm please?

Comment (ijuma) [reply build.gradle:120]: @niteshmor any build for a scala project will include a `zinc` configuration. The zinc configuration is only used during compilation, so I think what we have here is fine.

Comment (niteshmor) [reply build.gradle:120]: If that's the case, then this is fine. Thanks Ismael.
@rajinisivaram, you have my +1 for the PR as far `build.gradle` is concerned.

**Comment by rajinisivaram** [comment]
Thanks for the review @niteshmor @ijuma . Going to merge this one since the build is good and the conflicting part was reviewed.

**Pull request merged**